--- python-pip-9.0.1.orig/pip/__init__.py
+++ python-pip-9.0.1/pip/__init__.py
@@ -18,7 +18,10 @@ import re
 # to add socks as yet another dependency for pip, nor do I want to allow-stder
 # in the DEP-8 tests, so just suppress the warning.  pdb tells me this has to
 # be done before the import of pip.vcs.
-from pip._vendor.requests.packages.urllib3.exceptions import DependencyWarning
+try:
+  from pip._vendor.requests.packages.urllib3.exceptions import DependencyWarning
+except ImportError:
+  from urllib3.exceptions import DependencyWarning
 warnings.filterwarnings("ignore", category=DependencyWarning)  # noqa
 
 
@@ -29,10 +32,14 @@ from pip.vcs import git, mercurial, subv
 from pip.baseparser import ConfigOptionParser, UpdatingDefaultsHelpFormatter
 from pip.commands import get_summaries, get_similar_commands
 from pip.commands import commands_dict
-from pip._vendor.requests.packages.urllib3.exceptions import (
-    InsecureRequestWarning,
-)
-
+try:
+    from pip._vendor.requests.packages.urllib3.exceptions import (
+        InsecureRequestWarning,
+    )
+except ImportError:
+    from urllib3.exceptions import (
+        InsecureRequestWarning,
+    )
 
 # assignment for flake8 to be happy
 
